easybuild/easyconfigs/s/SciPy-bundle/SciPy-bundle-2023.07-iimkl-2023a.eb

```diff
@@ -0,0 +1,96 @@
+easyblock = 'PythonBundle'
+
+name = 'SciPy-bundle'
+version = '2023.07'
+
+homepage = 'https://python.org/'
+description = "Bundle of Python packages for scientific software"
+
+toolchain = {'name': 'iimkl', 'version': '2023a'}
+toolchainopts = {'pic': True, 'lowopt': True}
+
+builddependencies = [
+    ('hypothesis', '6.82.0'),
+    ('UnZip', '6.0'),
+    # scipy >= 1.9.0 uses Meson/Ninja
+    ('Meson', '1.1.1'),
+    ('Ninja', '1.11.1'),
+    ('pkgconf', '1.9.5'),   # required by scipy
+]
+
+dependencies = [
+    ('Python', '3.11.3'),
+    ('Python-bundle-PyPI', '2023.06'),
+    ('pybind11', '2.11.1'),  # required by scipy
+]
+
+use_pip = True
+
+# order is important!
+exts_list = [
+    ('numpy', '1.25.1', {
+        'patches': [
+            'numpy-1.18.2-mkl.patch',
+            'numpy-1.22.3_disable-broken-override-test.patch',
+        ],
+        'checksums': [
+            {'numpy-1.25.1.tar.gz': '9a3a9f3a61480cc086117b426a8bd86869c213fc4072e606f01c4e4b66eb92bf'},
+            {'numpy-1.18.2-mkl.patch': 'ea25ad5c0148c1398d282f0424e642fb9815a1a80f4512659b018e2adc378bcf'},
+            {'numpy-1.22.3_disable-broken-override-test.patch':
+             '9c589bb073b28b25ff45eb3c63c57966aa508dd8b318d0b885b6295271e4983c'},
```

Comment:
oh, I see now you excluded `numpy-1.20.3_disable_fortran_callback_test.patch` that is used in the previous intel builds.

Reply:
Yes, it was something that was common to both toolchains, and then removed from the gfbf/2023a build, so I assumed it was something that had been fixed upstream.  I'll try again with that one.

Reply:
Porting @akesandgren's patch to the new numpy source fixed this issue.  But there is a new one with compiling some C stuff in the tests.  I'll update the PR, and then post that when I have investigated a bit more...